src/main/kotlin/com/openlattice/edm/processors/GetEntityTypeFromEntitySetEntryProcessor.kt

```diff
@@ -0,0 +1,13 @@
+package com.openlattice.edm.processors
+
+import com.hazelcast.core.ReadOnly
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.EntitySet
+import java.util.*
+
+class GetEntityTypeFromEntitySetEntryProcessor : AbstractRhizomeEntryProcessor<UUID, EntitySet, Optional<UUID>>(), ReadOnly {
+    override fun process(entry: MutableMap.MutableEntry<UUID, EntitySet>?): Optional<UUID> {
```

Comment:
Pretty sure the generated signature is wrong and entry will never be null

Reply:
It can't be null given the way we've used it but used in other contexts, it could be null :'(

Reply:
Only the value can be null. The entry itself will never be null (at least according Hazelcast Docs). The signature be MutableEntry<UUID, EntitySet?> but I'm not sure there's a way for kotlin to know that.

Reply:
EntryProcessor:88
"     * The entry's value will be {@code null}, if the entry for the key doesn't exist."
Which, given the way we use it, should not happen but totally could happen

Reply:
Wait what
😕

Reply:
Okay, I see. Am fixing

Reply:
Related: https://github.com/geekbeast/rhizome/pull/282
and further related: #628